### ISSUE TITLE
Fixing for CS2

### ISSUE THREAD

**Issue by kubo6472** [timche/docker-csgo#39]
Apparently, now, the only way to make this work is with populating with own files, since 740 still says it's CS:GO DS.

**Comment by timche** [comment]
Can you share what files you need to add?

**Comment by timche** [comment]
> Can I run or play on community servers in Counter-Strike 2?
> Not at launch, but Community server support is coming soon.

https://help.steampowered.com/en/faqs/view/5ED2-ED8E-81F4-0C18#commserver

So we're still waiting for official support.

**Comment by kubo6472** [comment]
> Can you share what files you need to add?

If you copy over the 730 folder of regularly installed CS2, the srcds will fix it correctly and it will be playable

**Comment by kubo6472** [comment]
> My server say it is 730 without any change from my side: 2023-09-28 14:40:20 Setting breakpad minidump AppID = 730 I will check later if is indeed playable.

And version ID? CS:GO is 740:730

**Comment by GOATS2K** [comment]
> My server say it is 730 without any change from my side: 2023-09-28 14:40:20 Setting breakpad minidump AppID = 730 I will check later if is indeed playable.

Did it work?

**Comment by anton0k** [comment]
Without adjustment it does not work, sorry for the confusion.

**Comment by anton0k** [comment]
> Shouldn't it be enough to just switch [from CS:GO app id 740 to CS2 app id 730](https://github.com/timche/docker-csgo/blob/main/base/server.sh#L25) when installing/updating the server? If someone could validate or confirm please.

I changed the line but that did not do the trick.
Changing the other two lines to 730 gave me: 2023-09-29 09:43:42 ERROR! Failed to install app '730' (No subscription)

**Comment by GOATS2K** [comment]
Setting up a CS2 server is a bit different, I managed to get it to work today using `steamcmd`.

You'll need to fetch the CS2 game using app ID 730 and run it with a `-dedicated` flag. 

```
./cs2 -dedicated +map de_dust2 -usercon -console +mp_autokick 0
```

You cannot use an anonymous login to get the game.

**Comment by sazarkin** [comment]
@GOATS2K looks like it works, thanks. I've updated debian version and used `$server_dir/game/bin/linuxsteamrt64/cs2`. But cannot make rcon work ingame, looks like it's only exposed on localhost...

**Comment by GOATS2K** [comment]
Yeah, I cannot get rcon to work either. Glad you got the game to run!

**Comment by kubo6472** [comment]
> @GOATS2K looks like it works, thanks. I've updated debian version and used `$server_dir/game/bin/linuxsteamrt64/cs2`. But cannot make rcon work ingame, looks like it's only exposed on localhost...

Please PR if you can, thanks.
